app/containers/markdown/index.js

```diff
@@ -381,6 +381,7 @@ class Markdown extends PureComponent {
 		// Ex: '[ ](https://open.rocket.chat/group/test?msg=abcdef)  Test'
 		// Return: 'Test'
 		m = m.replace(/^\[([\s]]*)\]\(([^)]*)\)\s/, '').trim();
+		m = m.replace(/\n(?=\n)/g, '\n&nbsp;');
```

Comment:
I didn't had the chance to investigate the markdown, but it looks like this shouldn't be done in the component.
The problem is most likely to be on the markdown lib not parsing correctly.
Can you check?

Reply:
Yes, the problem definitely resided in the markdown library, in that it doesn't render empty lines, following from HTML standards. The problem here is that, even though that is acceptable behavior for the markdown library by its own standards, it introduces inconsistencies in Rocket Chat, being that these empty lines are acknowledged on the browser, but not in the app.
The change I made puts an empty space character in the empty lines so that markdown acknowledges them as would be, on the browser client. This also doesn't introduce any possible future problems, in case they change their policy on this.
One other solution could be to replace these particular **\n** breaks with <br> tags, instead of adding a whitespace character.

Reply:
@diegolmello What should I do?